content/_help/verify-your-identity/verify-your-identity-in-person._fr.md

```diff
@@ -0,0 +1,135 @@
+
+### Votre carte d'identité délivrée par l'État
+
+Apportez la même carte d'identité que vous avez utilisée sur Login.gov. Votre pièce d'identité ne doit pas être périmée. Pour l'instant, seules les pièces d'identité suivantes, délivrées par l'État, sont acceptées : 
+
+* Un permis de conduire des 50 États, du district de Columbia (D. C.) et des autres territoires américains (Guam, îles Vierges américaines, îles Mariannes et Porto Rico).
+* Une carte d'identité sans permis de conduire délivrée par l'État.
+* * Il s'agit d'un document d'identité délivré par l'État, le district de Columbia (D. C.) ou le territoire américain, qui atteste de l'identité mais ne donne pas le droit de conduire.
+
+Nous n'acceptons actuellement aucune autre forme d'identification, comme les passeports et les cartes d'identité militaires. Vous ne pouvez pas vérifier votre identité au bureau de poste sans un document d'identité délivré par l'État. Nous travaillons actuellement à l'ajout d'autres moyens de vérifier votre identité.
+
+En attendant, veuillez contacter le centre d'assistance de l'agence partenaire pour savoir ce que vous pouvez faire.
+
+### Preuve de votre adresse actuelle
+
+Si votre adresse actuelle ne correspond pas à l'adresse figurant sur votre pièce d'identité, vous devrez apporter une preuve de votre adresse actuelle au bureau de poste. Les formes acceptables de preuve d'adresse sont :
+
+* Bail, hypothèque ou acte de fiducie
+* Inscription sur les listes électorales
+* Carte d'immatriculation d’un véhicule
+* Police d'assurance habitation ou véhicule
+
+Vous pouvez apporter une copie physique ou montrer une copie numérique du document. Vous ne pouvez pas montrer une photo ou une capture d'écran du document. 
+
+## Effectuez le processus au bureau de poste avant la date limite
+
+### Sites disponibles dans la région de Washington D. C. 
+
+Vous pouvez vous rendre dans l'un des bureaux de poste suivants. Vous n'avez pas besoin de vous rendre à l'emplacement que vous avez sélectionné sur Login.gov. 
+
+1. BALTIMORE — Post Office™\
+   900 E FAYETTE ST RM 118\
+   BALTIMORE, MD 21233-9715
+2. BETHESDA — Post Office™\
+   6900 WISCONSIN AVE STE 100\
+   CHEVY CHASE, MD 20815-9996
+3. FRIENDSHIP — Post Office™\
+   4005 WISCONSIN AVE NW\
+   WASHINGTON, D. C. 20016-9997
+4. WASHINGTON — Post Office™\
+   900 BRENTWOOD RD NE\
+   WASHINGTON, D. C. 20018-9997
+5. ARLINGTON — Post Office™\
+   3118 WASHINGTON BLVD\
+   ARLINGTON, VA 22201-9998
+6. ASHBURN — Post Office™\
+   44715 PRENTICE DR\
+   DULLES, VA 20101-9996
+7. MERRIFIELD — Post Office™\
+   8409 LEE HWY\
+   MERRIFIELD, VA 22116-9998
```

Comment:
Same question about bolded names, for French. Also, these slashes "\\" will appear literally in the page content (it's [a Netlify CMS issue](https://github.com/netlify/netlify-cms/issues/512)).

```suggestion
1. **BALTIMORE — Post Office™**
   900 E FAYETTE ST RM 118
   BALTIMORE, MD 21233-9715
2. **BETHESDA — Post Office™**
   6900 WISCONSIN AVE STE 100
   CHEVY CHASE, MD 20815-9996
3. **FRIENDSHIP — Post Office™**
   4005 WISCONSIN AVE NW
   WASHINGTON, D. C. 20016-9997
4. **WASHINGTON — Post Office™**
   900 BRENTWOOD RD NE
   WASHINGTON, D. C. 20018-9997
5. **ARLINGTON — Post Office™**
   3118 WASHINGTON BLVD
   ARLINGTON, VA 22201-9998
6. **ASHBURN — Post Office™**
   44715 PRENTICE DR
   DULLES, VA 20101-9996
7. **MERRIFIELD — Post Office™**
   8409 LEE HWY
   MERRIFIELD, VA 22116-9998
```

Reply:
AH, thought I'd removed the slashes, thank you!